google/cloud/compute/firewall_policies/v1/internal/firewall_policies_rest_connection_impl.cc

```diff
@@ -92,10 +92,14 @@ FirewallPoliciesRestConnectionImpl::AddAssociation(
       },
       [request](std::string const& op,
                 google::cloud::cpp::compute::global_organization_operations::
-                    v1::GetOperationRequest& r) { r.set_operation(op); },
+                    v1::GetOperationRequest& r) {
+        r.set_operation(op);
+      },
```

Comment:
I'm not sure why these re-formattings are getting flipped back-and-forth, but, given that it is unrelated to adding the new library, I think I would revert this file and figure things out after.

Reply:
Missed this one sorry. I can revert tomorrow